src/main/modules/awilix/index.ts

```diff
@@ -27,21 +21,27 @@ const logger: LoggerInstance = Logger.getLogger('app');
  */
 export class Container {
   public enableFor(app: Application): void {
-    app.locals.container = createContainer({ injectionMode: InjectionMode.CLASSIC }).register({
+    const stepControllers = getSteps().map((step: Step) => {
```

Comment:
is that type annotation necessary? if getSteps returns Step[] it should be able to work it out

Reply:
👍 Good spot, fixed in d92f164.